rules.ini

```diff
@@ -53,7 +53,7 @@ CryEngine[] = (?:^|/)engine\.pak$
 CryEngine[] = \.cry$
 Defold = (?:^|/)game\.dmanifest$
 FNA = (?:^|/)fna\.dll$
-Godot = (?:^|/)(?:GODOT|libqodot\.dll$)
+Godot = (?:^|/)GODOT_LICENSE\.txt$
```

Comment:
Note that it's not a standard file that we produce on export, so if you found depots using this, the users created the file and named it this way themselves.

On the other hand a case insensitive match on `godot` could maybe be useful? Would match this license, or any arbitrary Godot-related file (e.g. `LICENSE.godot.md`, or `project.godot` if the project ships with source code).

Reply:
Thanks. i'm going to just go ahead and search all of steamdb to see how often the word 'godot' appears. If it's not common we'll probably just kill this rule if it's not emitted by the engine. The one game example I had that used this also would be caught by the new godot PR rules.